modules/web/src/app/project/template.html

```diff
@@ -374,11 +374,11 @@ <h2>Get Started</h2>
 
 <ng-template #allProjectToggle>
   <mat-slide-toggle *ngIf="isAdmin"
-                        class="all-projects-toggle"
-                        [disabled]="isProjectsLoading"
-                        labelPosition="before"
-                        [matTooltip]="isProjectsLoading ? 'Please wait while projects are loading' : null"
-                        [ngModel]="settings.displayAllProjectsForAdmin"
-                        (toggleChange)="changeProjectVisibility()">Show All Projects
+                    class="all-projects-toggle"
+                    [disabled]="isProjectsLoading"
```

Comment:
Do we know why these issues with indentations are not being caught by our pipelines? It seems like a common occurrence these days. Because these should have been already fixed in the last PR where these files were modified. 

I noticed this issue a few weeks ago as well.

An example from your own PR would be https://github.com/kubermatic/dashboard/pull/6144/files#diff-1f46f73009b38ea4ecfc48cd53babc9b1f45fb74a441dbab8cd538c1c1c8a190R134, CI should have complained about this.